packages/api/test/index.test.ts

```diff
@@ -90,19 +90,34 @@ describe('api', function () {
   describe('#accessors', function () {
     it('should have a function for each http method', function () {
       ['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace'].forEach(method => {
-        expect(petstoreSdk[method]).to.be.a('function');
+        expect(petstoreSDK[method]).to.be.a('function');
       });
     });
 
     describe('#operationId()', function () {
       it('should work for operationId', async function () {
         fetchMock.get(`${petstoreServerUrl}/pets`, mockResponses.real('it worked!'));
 
-        expect(await petstoreSdk.findPets()).to.equal('it worked!');
+        expect(await petstoreSDK.findPets()).to.equal('it worked!');
       });
 
-      it('should work with operationIds that have contain spaces', function () {
-        expect(petstoreSdk['find pet by id']).to.be.a('function');
```

Comment:
This test should have actually been failing but it wasn't because everything on `petstoreSDK` is a function... because that's how you call it. We only determine if `find by pet id` is actually an operationID until you call it, which we weren't doing here. Yikes!